template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Startup.cs

```diff
@@ -5,7 +5,6 @@
 #if (OrganizationalAuth || IndividualB2CAuth)
```

Comment:
using Microsoft.AspNetCore.Http is not referenced in Startup.cs. Although this was the case before you change